README.md

```diff
@@ -27,7 +26,7 @@ Based on draft-ietf-eppext-launchphase-07 / draft-ietf-regext-launchphase-00
 Based on draft-ietf-regext-epp-fees-06 (fee-0.23)
 
 ##### New gTLD backend providers (RO's)
-Afnic, Afilias, Amazon (Neustar), ARI (now Neustar), CoCCA, CentralNIC, CoreNIC, Charleston Road Registry, CNNIC, Donuts, Famous Four Media (Nominet), Fury Registry (CIRA), GMO Registry, KNET, KSRegistry, Minds And Machines (Nominet-MMX), Neustar, Nic.BR, Nic.MX, Nominet, OpenRegistry, Registry-in-a-Box from NicAT, Rightside, SIDN, Tango from KNIPP, TCI, TeleInfo, UniRegistry, Verisign, ZACR
+Afnic, Afilias, Amazon (GoDaddy DNRS, Former Neustar/ARI), CoCCA, CentralNIC, CoreNIC, Charleston Road Registry, CNNIC, Donuts, Famous Four Media (Nominet), Fury Registry (CIRA), GMO Registry, KNET, KSRegistry, Minds And Machines (Nominet-MMX), Nic.BR, Nic.MX, Nominet, OpenRegistry, Registry-in-a-Box from NicAT, Rightside, SIDN, Tango from KNIPP, TCI, TeleInfo, UniRegistry, Verisign, ZACR
```

Comment:
We can remove Famous Four Media and Minds and Machines (Nominet-MMX) references - i think that they are not in use anymore

Reply:
Just in this file? or whole Net::DRI? or even dddri? i think we can have a ticket for that if its more than just 1 or 2 files